qiskit/pulse/builder.py

```diff
@@ -1617,12 +1666,21 @@ def call_circuit(circ: circuit.QuantumCircuit):
     Args:
         Circuit to call.
     """
-    _active_builder().call_circuit(circ, lazy=True)
+    warnings.warn('``call_circuit`` is being deprecated. '
+                  '``call`` function can take both schedule and circuit.', DeprecationWarning)
+
+    call(circ)
 
 
 def call(target: Union[circuit.QuantumCircuit, Schedule]):
```

Comment:
Now that parameters are supported, I think it makes sense to be able to pass the parameters at call time here by name `pulse.call(sched, p1=0, ..., params=param_dict)`? Likewise, these should go in the `Call` instruction `Call(subroutine, consolidated_param_dict)`?